arduino/libraries/WiFi/src/WiFi.cpp

```diff
@@ -32,6 +32,12 @@
 #include <lwip/inet_chksum.h>
 
 #include "WiFi.h"
+#include "mdns.h"
+
+// defines required from mdns service
+#define LWIP_MDNS_RESPONDER             1
+#define LWIP_IGM                        1
+#define LWIP_IPV4                       1
```

Comment:
It seems really strange we need these defines to be manually added ... are you sure we need them?

Reply:
removed!! added on the kconfig